src/concurrency/transaction.cpp

```diff
@@ -65,6 +65,11 @@ Transaction::~Transaction() {}
 void Transaction::Init(const size_t thread_id,
                        const IsolationLevelType isolation, const cid_t &read_id,
                        const cid_t &commit_id) {
+  // get current time when Transaction is created
+  struct timeval tv;
+  gettimeofday(&tv, NULL);
```

Comment:
Since the timer is a new function, can you submit this to a new PR?

Reply:
I didn't modify so much to add the timestamp. So I just submitted to this PR. Should I open a new PR?

Reply:
@haoxianghua Yes, it would be wise to open a new PR for a new feature. What is the purpose of the timestamp?

Reply:
The timestamp is to make a snapshot of the current time when the txn begins so that when calling now() in build in function, it would return the time when this txn opened.

Reply:
Got you!